sensu/config.go

```diff
@@ -72,6 +73,15 @@ func NewConfigFromFile(flagset *configFlagSet, configFile string) (*Config, erro
 		return nil, errNoClientName
 	}
 
+	// If we reached this point, we have a client name.
+	// Let's also add the default subscription - client:name
+	// Without at least one subscription sensu server will crash.
+	cfg.config.Client.Subscriptions = append(cfg.config.Client.Subscriptions,
```

Comment:
hi @AlexisMontagne,
the requirement for at least one subscription comes from the sensu server crashing, but if we want to duplicate the behaviour of the ruby client we must **always** add this subscription `client:name`. I tested a config with other subscriptions and it appears like the ruby client still adds it.